src/app/zones/views/ZoneListView.vue

```diff
@@ -30,20 +45,44 @@
         <ZoneDetails :zone-overview="entity" />
       </div>
     </div>
+
+    <DeleteResourceModal
+      v-if="isDeleteModalVisible"
+      :confirmation-text="deleteZoneName"
+      :delete-function="deleteZone"
+      :is-visible="isDeleteModalVisible"
+      modal-id="delete-zone-modal"
+      :action-button-text="i18n.messages.zones.delete.confirmModalProceedText"
+      :title="i18n.messages.zones.delete.confirmModalTitle"
+      @cancel="toggleDeleteModal"
+      @delete="handleDelete"
+    >
+      <template #body-content>
+        <p>{{ i18n.messages.zones.delete.confirmModalText1 }}</p>
+
+        <p>{{ i18n.messages.zones.delete.confirmModalText2 }}</p>
```

Comment:
it might be already that it does. Take a look at: 
![image](https://github.com/kumahq/kuma-gui/assets/4562608/8d78dd1a-921c-4d3f-98bf-b43a2ae97326)

Reply:
Note that this PR is only adding a minimal effort interface for the i18n layer while not integrating with any dedicated library for internationalization, yet. So as the utilities present in this PR stand, they can’t do that.